openedx/core/djangoapps/users/user_tags/provider.py

```diff
@@ -0,0 +1,65 @@
+"""
+Abstract base class for all User Tag Providers.
+"""
+
+
+class UserTagProvider(object):
+    """
```

Comment:
@efagin FYI on this PR.  I'll be closing it now.  But we can go back to this if/when ever we want to revisit User Tagging.